apollo-core/src/main/java/com/ctrip/framework/foundation/internals/NetworkInterfaceManager.java

```diff
@@ -119,15 +119,13 @@ private void load() {
 
     try {
       Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
-      List<NetworkInterface> nis = interfaces == null ? Collections.emptyList()
-          : Collections.list(interfaces);
+      if (interfaces == null) {
+        m_local = InetAddress.getLoopbackAddress();
+        return;
+      }
+      List<NetworkInterface> nis = Collections.list(interfaces);
```

Comment:
After reading the logic of this method, I think this by-pass logic is correct. But I'm wondering how about we simply leave the original logic as it is?
```java
List<NetworkInterface> nis = interfaces == null ? Collections.emptyList()
          : Collections.list(interfaces);
```

Reply:
The original code is fine, the new code should be fine, and the updated code performs better